notebooks/ingestion/Loading_DataModels.ipynb

```diff
@@ -4,27 +4,79 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Loading the data into the DB, using Data Models (1/3).\n",
+    "# Ingest Cookbook (DataModels).\n",
     "\n",
-    "Following the concepts on the different [means](/HowToGuides/start/Ingestion) of ingesting the data, we will build an example using the DataModel method in this notebook.\n",
+    "Following the concepts on the different [means](https://docs.aperturedata.io/HowToGuides/Ingestion/Ingestion#ingestion-based-on-the-data-model) of ingesting the data, we will build an example using the DataModel method in this notebook.\n",
     "\n",
     "We will use the [Cookbook dataset](https://docs.google.com/spreadsheets/d/1G1HPG3Dxx5W39OD6b74wMHvWupD7N-DLUbV7tD5owx8/edit?usp=sharing) to have the data persisted onto ApertureDB instance.\n",
     "\n",
-    "We merge the first 3 sheets on the source into a json file such that there will be a list of Dishes objects, and each Dish may have multiple ingredients, along with some oher miscellaneous properties."
+    "> #### Note\n",
+    "> The notebook assumes that the SDK has been setup as per the instructions.\n",
+    ">\n",
+    "> Colab one time setup [instructions](/Setup/client/notebooks?notebooks=colab).\n",
+    ">\n",
+    "> Non ephemeral jupyter server one time setup [instructions](/Setup/client/notebooks?notebooks=notebook).\n",
+    ">\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Data Models\n",
+    "## Needed resources.\n",
+    "- ```create_nested_json.py```: The script merges the first 3 sheets on the source into a json file such that there will be a list of Dishes objects, and each Dish may have multiple ingredients, and each ingredient has miscellaneous properties. This ends up as a json file.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-10-01 16:33:55--  https://github.com/aperture-data/Cookbook/raw/refs/heads/main/scripts/create_nested_json.py\n",
+      "Resolving github.com (github.com)... 140.82.112.3\n",
+      "Connecting to github.com (github.com)|140.82.112.3|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 Found\n",
+      "Location: https://raw.githubusercontent.com/aperture-data/Cookbook/refs/heads/main/scripts/create_nested_json.py [following]\n",
```

Comment:
Clear out download / log output -- especially since it seems to put your local machine info

Reply:
will do